### DISCUSSION TITLE
Installing Elementary OS 7.1 on a Victus 15" (2022)

### DISCUSSION THREAD

**Discussion by golfballthegolfster** [elementary/installer#759] [Q&A]
Hey there, Elementary.

I was trying to install the OS the first time, since all my data was wiped on the PC, but then I keep getting this message.
![17125777257756629493811943760171](https://github.com/elementary/installer/assets/155264247/2ce9eadc-8637-464f-b6f0-12c860dfbc7c)
Someone help me here?

**Comment by golfballthegolfster** [comment]
![17125778174593291738100257335323](https://github.com/elementary/installer/assets/155264247/93eda9ff-e9c7-45d6-bd7c-e83b3fce85d0)

**Reply by danirabbit** [reply]
LVM partitions might be locked in which case you won't be able to write over them.

On the custom install screen, select "Modify Partitions". When Gparted opens, look for a keyhole symbol next to one of your partitions. If it's present, you can right click on that partition and there should be an option like "deactivate". Doing that should make the partition write able and you can try installing again.

If it still fails for some reason, try manually deleting that LVM partition before installing
